extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -114,9 +72,7 @@ function load_assets( $attr ) {
 					method="post"
 					id="mailchimp_form"
 					target="_top"
-					<?php if ( $is_amp_request ) : ?>
 					on="submit-success:AMP.setState( { mailing_list_status: 'subscribed', mailing_list_email: event.response.email } )"
-					<?php endif; ?>
```

Comment:
Uh, nice catch here!
I was almost all like _HEY YOU! Why did you remove this?_ but gladly I double-checked first! 😄